weaver/tranform/transform.py

```diff
@@ -0,0 +1,379 @@
+import base64
+import csv
+import json
+import os.path
+import shutil
+import tarfile
+import tempfile
+
+import jinja2
+import pandas as pd
+import xmltodict
+import yaml
+from PIL import Image
+from bs4 import BeautifulSoup
+from cairosvg import svg2png
+from celery.utils.log import get_task_logger
+from fpdf import FPDF
+from json2xml import json2xml
+from json2xml.utils import readfromjson
+from pyramid.httpexceptions import HTTPUnprocessableEntity
+from pyramid.response import FileResponse
+
+from weaver.formats import get_extension
+from weaver.tranform.png2svg import rgba_image_to_svg_contiguous
+from weaver.tranform.tiff import Tiff
+
+from weaver.tranform.utils import is_png, is_gif, is_svg, write_content, get_content, is_image, is_tiff, \
+    get_file_extension
+
+LOGGER = get_task_logger(__name__)
+
+HTML_CONTENT = """<html>
+    <head></head>
+    <body><p>%CONTENT%</p></body>
+    </html>"""
+
+
+FAMILIES = [
+    ["text/plain", "text/html", "application/pdf"],
+    ["image/png", "image/gif", "image/jpeg", "image/tiff", "image/svg+xml", "application/pdf"],
+    ["text/csv", "application/xml", "application/x-yaml", "application/json"]
+]
```

Comment:
I think those should be defined in `weaver/formats.py` and reuse the media-type references defined in there.